packages/devtools_app/lib/src/vm_developer/vm_statistics_view.dart

```diff
@@ -0,0 +1,304 @@
+          MapEntry(
+            'Max Memory (RSS)',
+            prettyPrintBytes(
+              vm?.maxRSS,
+              includeUnit: true,
+            ),
+          ),
+          MapEntry(
+            'Current Memory (RSS)',
+            prettyPrintBytes(
+              vm?.currentRSS,
+              includeUnit: true,
+            ),
+          ),
+          MapEntry(
+            'Zone Memory',
+            prettyPrintBytes(
+              vm?.nativeZoneMemoryUsage,
+              includeUnit: true,
+            ),
+          ),
+        ],
+      ),
+    );
+  }
+}
+
+class _IsolateNameColumn extends ColumnData<Isolate> {
+  _IsolateNameColumn() : super('Name');
+
+  @override
+  String getValue(Isolate i) => i.name;
+}
+
+class _IsolateIDColumn extends ColumnData<Isolate> {
+  _IsolateIDColumn() : super('ID');
+
+  @override
+  String getValue(Isolate i) => i.number;
+}
+
+abstract class _IsolateMemoryColumn extends ColumnData<Isolate> {
+  _IsolateMemoryColumn(String title) : super(title);
+
+  int getCapacity(Isolate i);
+  int getUsage(Isolate i);
+
+  @override
+  String getValue(Isolate i) {
+    return '${prettyPrintBytes(getUsage(i), includeUnit: true)} '
+        'of ${prettyPrintBytes(getCapacity(i), includeUnit: true)}';
+  }
+}
+
+class _IsolateNewSpaceColumn extends _IsolateMemoryColumn {
+  _IsolateNewSpaceColumn() : super('New Space');
+
+  @override
+  int getCapacity(Isolate i) => i.newSpaceCapacity;
+
+  @override
+  int getUsage(Isolate i) => i.newSpaceUsage;
+}
+
+class _IsolateOldSpaceColumn extends _IsolateMemoryColumn {
+  _IsolateOldSpaceColumn() : super('Old Space');
+
+  @override
+  int getCapacity(Isolate i) => i.oldSpaceCapacity;
+
+  @override
+  int getUsage(Isolate i) => i.oldSpaceUsage;
+}
+
+class _IsolateHeapColumn extends _IsolateMemoryColumn {
+  _IsolateHeapColumn() : super('Heap');
+
+  @override
+  int getCapacity(Isolate i) => i.dartHeapCapacity;
+
+  @override
+  int getUsage(Isolate i) => i.dartHeapSize;
+}
+
+/// Displays general statistics about running isolates including:
+///   - Isolate debug name
+///   - VM Isolate ID
+///   - New / old space usage
+///   - Dart heap usage
+class IsolatesPreviewWidget extends StatelessWidget {
+  IsolatesPreviewWidget({
+    @required this.controller,
+    this.systemIsolates = false,
+  });
+
+  final name = _IsolateNameColumn();
+  final number = _IsolateIDColumn();
+  final newSpace = _IsolateNewSpaceColumn();
+  final oldSpace = _IsolateOldSpaceColumn();
+  final heap = _IsolateHeapColumn();
+
+  List<ColumnData<Isolate>> get columns => [
+        name,
+        number,
+        newSpace,
+        oldSpace,
+        heap,
+      ];
+
+  final VMStatisticsViewController controller;
+  final bool systemIsolates;
+
+  @override
+  Widget build(BuildContext context) {
+    final title = systemIsolates ? 'System Isolates' : 'Isolates';
+    final isolates =
+        systemIsolates ? controller.systemIsolates : controller.isolates;
+    return Flexible(
```

Comment:
same - not trying to beat a dead horse, but just pointing out all the spots so you don't have to hunt :)